### DISCUSSION TITLE
Copilot is Encountering Temporary Issues

### DISCUSSION THREAD

**Discussion by michaelhvisser** [community/community#39264] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have attempted to uninstall, reinstall, both the Copilot Extension for VS Code and the Nightly Extension and both provide the same error "copilot is experiencing temporary issues". This issue has persisted for over 24 hours. The system status page on Github shows that copilot is up and running. Not sure what else I need to do.

I have paid for a full one year subscription and authorized the extension.

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by michaelhvisser** [comment, marked as answer]
Once I was able to get into the logs I found out it was related to my employers VPN. I have disabled it and now I am getting suggestions. Thanks!
